scripts/ci/test_index.py

```diff
@@ -106,14 +106,16 @@ def test_filename_duplicates(self):
     @unittest.skipUnless(os.getenv('CI'), 'Skipped as not running on CI')
     def test_checksums(self):
         for exts in self.index['extensions'].values():
-            for item in exts:
-                ext_file = get_whl_from_url(item['downloadUrl'], item['filename'],
-                                            self.whl_cache_dir, self.whl_cache)
-                computed_hash = get_sha256sum(ext_file)
-                self.assertEqual(computed_hash, item['sha256Digest'],
-                                 "Computed {} but found {} in index for {}".format(computed_hash,
-                                                                                   item['sha256Digest'],
-                                                                                   item['filename']))
+            # only test the latest version
+            item = max(exts, key=lambda ext: LooseVersion(ext['metadata']['version']))
```

Comment:
`distutils.version.LooseVersion` is legacy and conformant only to the superseded [PEP 386](https://www.python.org/dev/peps/pep-0386/) (by [PEP 440](https://www.python.org/dev/peps/pep-0440/)). Perhaps we can migrate to the latest [`packaging.version`](https://packaging.pypa.io/en/latest/version.html).

See https://stackoverflow.com/a/11887885/2199657

Reply:
```py
>>> from distutils.version import LooseVersion, StrictVersion
>>> StrictVersion('1.0.0a1') < StrictVersion('1.0.0')
True
>>> LooseVersion('1.0.0a1') < LooseVersion('1.0.0')
False
```